tasks/update_last_login.rb

```diff
@@ -0,0 +1,16 @@
+require 'date'
+
+task :update_last_login, [:date] do |_, args|
+  args.with_defaults(date: Date.today.to_s)
+
+  if args[:date] == 'yesterday'
```

Comment:
This task accepting 'yesterday' for convenience?
Think that is going to be a common use case?

Reply:
Yes. I'm intending to run this after midnight, so it's a convenience to update the previous days sessions.

We could break this out into a different task, however?

Reply:
This is now a separate task